### ISSUE TITLE
Indentation is not enforced for collection initializers with .editorconfig

### ISSUE THREAD

**Issue by jez9999** [dotnet/roslyn#46667]
My `.editorconfig` file has:

```
[*.cs]
dotnet_diagnostic.IDE0055.severity = warning
# 4-space indentation
indent_style = space
indent_size = 4
```

Yet this `.cs` file code does not produce a warning:

```
            var testObj = new List<string> {
    "Foo",
    "Bar"
            };
```

**Comment by sharwell** [comment]
Duplicate of #8269
